src/GameLogic/TurnRunner.ts

```diff
@@ -15,7 +15,7 @@ export class TurnRunner {
 		this._gameSession = gameSession;
 	}
 
-	public runTurn(playerInput: PlayerAction, level: Level): TurnState {
+	public runTurn(playerInput: PlayerAction, session: GameSession, level: Level): TurnState {
```

Comment:
Do we need to pass in the level AND the game session? The game session already has the level.

Reply:
I am trying to think if there could ever be a situation where we pass one GameSession and a different Level and boy I sure hope it never happens, so let me refactor it